src/main/java/com/kwezal/bearinmind/core/auth/service/AuthService.java

```diff
@@ -43,10 +51,21 @@ public LoginResponseDto logIn(HttpServletResponse response, CredentialsDto crede
             throw incorrectCredentialsException(username, password);
         }
 
+        return logIn(response, user, authClient);
+    }
+
+    public LoginResponseDto logIn(final HttpServletResponse response, final User user, final AuthClient authClient) {
```

Comment:
Looks like this method can be private and could you move it below the signUp method?

Reply:
Good catch! Thanks, will fix.